packages/mobile/e2e/scripts/utils.ts

```diff
@@ -0,0 +1,32 @@
+import { E2E_TEST_WALLET } from './consts'
+export const Web3 = require('web3')
+export const ContractKit = require('@celo/contractkit')
+export const dotenv = require('dotenv')
+export const web3 = new Web3('https://alfajores-forno.celo-testnet.org')
+export const kit = ContractKit.newKitFromWeb3(web3)
+
+export const balanceError = async (address = E2E_TEST_WALLET, minBalance = 10) => {
+  let balanceObject = await getBalance(address)
+  for (const balance in balanceObject) {
+    if (balanceObject[balance as keyof typeof balanceObject] < minBalance) {
+      throw new Error(
+        `${balance} balance of ${address} is below ${minBalance}. Please refill from the faucet https://celo.org/developers/faucet 🙏`
+      )
+    }
+  }
+}
+export const getBalance = async (address = E2E_TEST_WALLET) => {
```

Comment:
```suggestion
export async function getBalance(address) {
```

Reply:
Fixed in https://github.com/valora-inc/wallet/pull/2325/commits/2a72d8204532a885b1fb40b95a45f1c7ce9f3fef!